pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/Utils.groovy

```diff
@@ -442,6 +454,7 @@ public class Utils {
      * @return The list of instances. May be empty.
      */
     @Nonnull
+    @Restricted(NoExternalUse.class)
```

Comment:
Normally there is no need to restrict package protected methods, but shouldn't hurt. But there is definitely no need to restrict a private method.
otoh this is groovy, so private/public modifiers are just for show :)

Reply:
Or no modifier in Groovy means `public`? I keep forgetting that :)
Anyway it's a good marker to have so others know to stfo ;)